python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -26,10 +26,13 @@ from cuml.solvers import QN
 from cuml.common.base import Base
 from cuml.metrics.accuracy import accuracy_score
 from cuml.utils import input_to_dev_array
+from cuml.utils.cupy_utils import checked_cupy_unique
```

Comment:
the error of CI comes from reintroducing this `import` since this function was deleted by #1532

Reply:
Ah, bad merge, thanks for catching!